app/controllers/view_components_controller.rb

```diff
@@ -4,4 +4,10 @@
 
 class ViewComponentsController < Rails::ApplicationController # :nodoc:
   include ViewComponent::PreviewActions
+
+  if Rails.env.test?
+    def system_test_entrypoint
+      render file: "./tmp/view_components/#{params.permit(:file)[:file]}", status: 200
```

Comment:
It's a nitpick, but isn't `status: 200` the default?

Reply:
Yup! I think I can remove it since it isn't really providing any value.